web/src/pages/indicators/[slug]/index.page.tsx

```diff
@@ -149,22 +165,60 @@ export default function IndicatorsDetailsPage({
 
 				<GridItem
 					cols={12}
-					md={hasLeftColumn ? 8 : 12}
-					lg={hasLeftColumn ? 9 : 12}
-					elementType="section"
+					sm={hasLeftColumn ? 7 : 12}
+					md={hasLeftColumn ? 9 : 12}
+					elementType={"section"}
 				>
-					{product.summary ? (
-						<div
-							dangerouslySetInnerHTML={{ __html: product.summary }}
-							className={styles.summary}
-						/>
-					) : isTempWithdrawn ? (
-						<div
-							dangerouslySetInnerHTML={{ __html: product.withdrawnNotes }}
-							className={styles.summary}
-						/>
-					) : null}
-					<PublicationsPrevNext chapters={chapters} />
+					<Grid gutter="loose">
+						<GridItem cols={12} md={8}>
+							<Grid gutter="loose">
+								<GridItem cols={12}>
+									{product.summary ? (
+										<div
+											dangerouslySetInnerHTML={{ __html: product.summary }}
+											className={styles.summary}
+										/>
+									) : isTempWithdrawn ? (
+										<div
+											dangerouslySetInnerHTML={{
+												__html: product.withdrawnNotes,
+											}}
+											className={styles.summary}
+										/>
+									) : null}
+
+									<EndorsingOrganisations
+										endorsingList={product.endorsementList}
+										productTypeName={product.productTypeName}
+									/>
+									<SupportingOrganisations
+										supportingList={product.supportingList}
+										productTypeName={product.productTypeName}
+									/>
+
+									<LogosList
+										logosList={product.additionalAuthorList}
+										productId={product.id}
+										logoType="author"
+									/>
+
+									<LogosList
+										logosList={product.accreditationList}
+										productId={product.id}
+										logoType="accreditation"
+									/>
+								</GridItem>
+							</Grid>
+						</GridItem>
+
+						{hasRelatedProduct ? (
+							<RelatedProducts relatedProducts={relatedqs} />
+						) : null}
```

Comment:
As per the comment in RelatedProduct.tsx - could we nest this component in the grid here rather than inside the component.